### DISCUSSION TITLE
How do I use custom lambda connector with one of my database fields in V3

### DISCUSSION THREAD

**Discussion by otcyury** [hasura/graphql-engine#10621] [Question] [answered]
Hi,

I created custom lambda connector in Python and added it to the graph. I can see if in the console and able to test and use it stand alone. I now need to use it with one of my DB fields, so I figured I need to create a relationship to a "surrogate" field. The same way as it's done in Orders deliveryDateFormatted, based on deliveryDate on Hasura's tutorial GraphQL page.

I have been doing everything via CLI since I can't make changes in browser console. How do I go about adding this relationship in my case?

Thanks

**Comment by robertjdominguez** [comment, marked as answer]
Hey, @otcyury — sorry for the delay!

Check out this newly-published [docs section](https://hasura.io/docs/3.0/data-modeling/relationship#between-a-model-and-a-command) for the low-down on how to connect a model to your Python function 🎉 

If you have other questions, throw them here. It's a bit tricky the first time, but very easy to pick up with the [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=HasuraHQ.hasura) to assist in authoring these relationships.
